reactor-core/build.gradle

```diff
@@ -211,14 +188,15 @@ task testNoMicrometer(type: Test, group: 'verification') {
 check {
   dependsOn testStaticInit
   dependsOn testNoMicrometer
-  if (System.env.TRAVIS != "true") {
+  if (!detectedCiServers.contains("TRAVIS")) {
 	dependsOn testNG
   }
   if (!version.endsWith('BUILD-SNAPSHOT')) {
 	dependsOn loops
   }
 }
 
+//TODO all java9 / stubs / java-specific stuff should go in a convention plugin ?
```

Comment:
with `buildSrc` plugins, the goal was to eliminate most advanced logic from build.gradle files. But since this is only applicable to `reactor-core` module, I guess we can keep it, as ugly as it is 🤔